### ISSUE TITLE
L1 loss bug

### ISSUE THREAD

**Issue by oriern** [wayaai/GAN-Sandbox#3]
Hi!
I think you have a bug in cGAN:
line 305:
combined_loss = np.add(combined_model.train_on_batch(edge_image_batch, [**edge_image_batch**, y_real]),
                                   combined_loss)

The L1 loss should be between generated image and ground truth(real image), and not "edge_image_batch".
